core/trino-server-rpm/src/test/java/io/trino/server/rpm/ServerIT.java

```diff
@@ -165,11 +165,11 @@ public void testRpmMetadata()
             assertThat(rpmMetadata).extractingByKey("Name").isEqualTo("trino-server-rpm");
             assertThat(rpmMetadata).extractingByKey("Epoch").isEqualTo("0");
             assertThat(rpmMetadata).extractingByKey("Release").isEqualTo("1");
-            assertThat(rpmMetadata).extractingByKey("Version").isEqualTo("440.SNAPSHOT"); // TODO: fixme
+            assertThat(rpmMetadata).extractingByKey("Version").isEqualTo("440-SNAPSHOT");
```

Comment:
This needs to come from a property or something

Reply:
It does come from the property.